File: Dataset/Leetcode/train/19/95.cpp
 class Solution {
public:
    ListNode* XXX(ListNode* head, int n) {
        ListNode* dummy = head;
        ListNode* first = head;
        ListNode* second =head;
        for (int i = 0; i < n; ++i) {
            first = first->next;
        }
        if(first==nullptr){
            dummy=dummy->next;
            return dummy;
        }
        while (first->next!=nullptr) {
            first = first->next;
            second = second->next;
        }
        second->next = second->next->next;
        return dummy;
    }
};

File: Dataset/Leetcode/train/9/95.java
class Solution {
    public boolean XXX(int x) {
        String str = Integer.toString(x);
        int l = 0;
        int r = str.length() - 1;

        while(l < r){
            if(str.charAt(l) != str.charAt(r)){
                return false;
            }


            l++;
            r--;
        }

        return true;
    }
}

